src/rpm/package.rs

```diff
@@ -142,12 +191,10 @@ impl RPMPackage {
     ///
     ///
     #[cfg(feature = "signature-meta")]
-    pub fn verify_signature<V>(&self, verifier: V) -> Result<(), RPMError>
+    pub fn verify_signature<V>(&self, verifier: V) -> Result<SignatureVerificationOutcome, RPMError>
```

Comment:
@drahnr I'm confused by this. Nothing actually returns `SignatureVerificationOutcome::Fail` so it doesn't seem useful? Even if it did, encoding an error condition in the success path feels very wrong.

Reply:
That's correct, fix eta today